src/main/java/racing/domain/Winners.java

```diff
@@ -0,0 +1,26 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Winners {
+    private List<Car> cars;
```

Comment:
```suggestion
    private Cars cars;
```
`Cars` 객체를 활용해 볼 수 있을까요?

그러면 아래의 `winners()` 의 로직도 객체에 메시지를 보내 객체가 일하도록 해보세요